workflows/phylogenetics/wf_snippy_streamline.wdl

```diff
@@ -0,0 +1,101 @@
+version 1.0
+
+import "../../workflows/standalone_modules/wf_snippy_variants.wdl" as snippy_variants_workflow
+import "../../workflows/phylogenetics/wf_snippy_tree.wdl" as snippy_tree_workflow
+import "../../tasks/phylogenetic_inference/task_centroid.wdl" as centroid_task
+import "../../tasks/phylogenetic_inference/task_referenceseeker.wdl" as referenceseeker_task
+import "../../tasks/task_versioning.wdl" as versioning
+
+# input is arrays
+# centroid takes in an array of assemblies
+# reference seeker takes in one sample -- result from centroid
+# get reference seeker genome
+# scatter all input read arrays to snippy_variants
+# gather outputs of snippy_variants as input to snippy_tree to run with reference seeker genome
+
+workflow snippy_streamline {
+  input {
+    Array[File] read1
+    Array[File] read2
+    Array[File] assembly_fasta
+    Array[String] samplenames
+    String tree_name
+  }
+
+  call centroid_task.centroid {
+    input:
+      assembly_fasta = assembly_fasta
+  }
+  call referenceseeker_task.referenceseeker {
+    input:
+      assembly_fasta = centroid.centroid_genome_fasta_file,
+      samplename = centroid.centroid_genome_fasta_filename
+  }
+  # call ncbi_download {
+  #   # add once merged 
+  # }
+
+  # see https://github.com/openwdl/wdl/issues/279 for syntax explanation
+  # see also https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#arraypairxy-ziparrayx-arrayy for zip explanation
+  scatter (triplet in zip(zip(read1, read2), samplenames)) {
+    call snippy_variants_workflow.snippy_variants_wf {
+      input:
+        read1 = triplet.left.left, # access the left-most object (read 1)
+        read2 = triplet.left.right, # access the right-side object on the left (read 2)
+        reference = ncbi_download.reference, 
```

Comment:
wanted to note that this line will likely need to be changed to a `select_first` to allow the user to provide an input genome or allow the ncbi-datasets task to provide one as input